supervisor/api/supervisorhttp/parser_test.go

```diff
@@ -0,0 +1,128 @@
+package supervisorhttp
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// parse parses the command from JSON into a "command"
+// struct with a check for success.
+func parse(t *testing.T, jsonByte []byte, cmd *command) {
+	err := parseCommand(bytes.NewReader(jsonByte), cmd)
```

Comment:
BTW, you can use 

```go
assert := assert.New(t)
assert.Nilf(err, `Can't parse a command: "%v".`, err)
```

.. and so on (without `t` in wvwry assert).

Reply:
ok